docker/docker_config.sh

```diff
@@ -5,6 +5,14 @@ DOCKER_IMAGE_NAME="ghcr.io/lasp/adamant:example-latest"
 export DOCKER_CONTAINER_NAME
 export DOCKER_IMAGE_NAME
 
+ON_LINUX=""
+case "$OSTYPE" in
+  linux*)
+    ON_LINUX="--add-host=host.docker.internal:host-gateway"
+    ;;
+esac
+export ON_LINUX
+
```

Comment:
OK I think the way you did this here is fine, but setting ON_LINUX to the "--add-host" option directly is very specific to its usage in the create_container script. So I vote we just move this case statement to create_container.sh, and remove it from here. If we need to do some other platform specific stuff in the future that affect multiple scripts, we can move some of the logic back into docker_config.sh.

Reply:
That makes sense! See new commits (tested on Manjaro Vulcan 23.1.0)